tests/mmm/test_transformers.py

```diff
@@ -148,6 +149,21 @@ def test_geometric_adstock_good_alpha(self, x, alpha, l_max):
         assert y_np[1] == x[1] + alpha * x[0]
         assert y_np[2] == x[2] + alpha * x[1] + (alpha**2) * x[0]
 
+    @pytest.mark.parametrize(
+        "x, alpha, l_max",
```

Comment:
Is the `x` important here? can it be the same for all the tests? Would be less moving parts 😄

Reply:
I basically copied the values from `test_geometric_adstock_good_alpha`. 
Let me just test an int and a float smaller and greater than the bounds. `x = np.ones(shape=(100)` for all of them